### DISCUSSION TITLE
When should we split a verse into ab and cd?

### DISCUSSION THREAD

**Discussion by wujastyk** [sushrutaproject1/Sushrutasamhita_Saktumiva1#61] [Ideas]
## The normal situation

When we transcribe from the manuscripts, we assign a unique ID to each piece of text.  This is the “xml:id”.  The general rule is that we use the numbering of the vulgate edition (Acarya 1938) as our xml:id. 

Thus, a line (`<l>`) might be transcribed like this: 

```xml
<l xml:id="SS.5.1.19"> jānīyād viṣadātāram ebhir liṅgaiś ca buddhimān | <caesura/>
                       vivakṣur muhyate pṛṣṭo nottaraṃ pratipadyate |</l>
```

where SS = Suśrutasaṃhitā, and 5.1.19 = Kalpasthāna, adhyāya 1, verse 19.

The same xml:id should be used across all the manuscript transcriptions for the śloka beginning “jānīyād” and ending “pratipadyate” and its variant forms.

When Saktumiva collates the electronic files, it looks for  the text labelled “SS.5.1.19” in each file and collates them.  That’s how it knows what chunks of text are meant to be compared.

## What if one of the witnesses does not have half a śloka? 

Take the example of verse 13 in the vulgate: 

```xml
<l xml:id="SS.5.1.13"> vikakṣasṛṣṭasaṃsṛṣṭaṃ savitānaṃ kṛtārcanam |
                       parīkṣitastrīpuruṣaṃ bhavec cāpi mahānasam || </l>
```

When we look at the manuscripts, we find that the first half of the verse is not present:

```xml
 <l xml:id="SS.5.1.13"> 
                        parīkṣitastrīpuruṣaṃ bhavec cāpi mahānasam || </l>
```

We could leave this as it is. Saktumiva would collate the chunks of 5.1.13 text against each other and the apparatus would show this result:

![image-20211011090652179](https://user-images.githubusercontent.com/762246/136821754-bdc225d3-c937-4fc6-9da9-c47fc3da407b.png)

The apparatus says that A has added the first half of the verse.   

If we split 13 into two halves in the vulgate file:

```xml
<l xml:id="SS.5.1.13ab"> vikakṣasṛṣṭasaṃsṛṣṭaṃ savitānaṃ kṛtārcanam |</l>
<l xml:id="SS.5.1.13cd"> parīkṣitastrīpuruṣaṃ bhavec cāpi mahānasam || </l>
```

And then record that ab is not present in the manuscript witnesses like this:

```xml
<l xml:id="SS.5.1.13ab"/>                
<l xml:id="SS.5.1.13cd">parīkṣitastrīpuru<cb/>ṣaṃ bhavec cāpi mahānasaṃ |</l>
```

Saktumiva will display this:
![image-20211011091514766](https://user-images.githubusercontent.com/762246/136821571-d1082ccf-37c8-4bc7-9ac8-9f98bda1f43f.png)

It may not look very different.  But the apparatus shows not that A has added something to verse 13, but rather that A has something at this point in the text that is not present in the Nepalese manuscripts. 

This is a subtle distinction.  Is it worth making?  I think so.  It captures the difference between the assumption that the Nepalese verses have something missing, and the assumption that the vulgate has pieced of text inserted here and there.  But I have some uncertainty about whether this is the right approach, and I would welcome discussion

## Verse insertions and reordering

But splitting verses becomes essential in cases where half-verses get rearranged into different sequences, or when new verses get inserted in the manuscripts between the two halves of a verse in the vulgate.  In those cases, we have no choice but to split things up; it’s the only way to represent the transmission.  

So, here's a slightly complicated case where the Nepalese manuscripts have material that comes between the vulgate's verse 5.3.35ab and 5.3.35.cd.  We call this material 5.3.35.1 and 5.3.35.2:  

```
<l xml:id="SS.5.3.35ab">muhūrttāt tad upādeyam prahāran daṃśavarjjitaṃ ||</l>
<l xml:id="SS.5.3.35.1"><add place="inline"><unclear>śī</unclear>lakṣate garbbhiṇikuṣṭhim ehi
                            <caesura/>
                             rūkṣeṣu<lb n="6"/> deheṣv avaleṣu caiva |</add>
                    </l>
<l xml:id="SS.5.3.35.2" sameAs="#SS.5.3.39.2">
                        <add place="inline"><unclear>raukṣyāt air ślakṣṇyā</unclear>d vikāsitvāt
                            tathaiva ca ||</add>
                        <caesura/>
                         viṣam atairgguṇairyuktaṃ kṣate samanudhāvati |
                        <caesura/>
                         vātātapābhyān nihataṃ nirvvīyam upajāyate ||
                        <caesura/>
                         tasmād vi<pb n="333r"/>ṣahataṃ sarvvaṃ bhakṣitan tu na mārayet || </l>

<l xml:id="SS.5.3.35cd">savātaṃ gṛhadhūmābhaṃ purīṣaṃ yo 'tisāryate |</l>
```
This is a case where it's much clearer and more true to the manuscripts to split 5.3.35 into ab and cd.  
And why, I hear you cry, do we have two insertions, SS.5.3.35.1 and SS.5.3.35.2?  Because one manuscript has them both, but the other manuscript only has one of them

**Comment by wujastyk** [comment]
As I reflect on this, I am getting less convinced about the need to split verses _except_ in cases where there are intrusions or re-orderings.  The first example from Saktumiva looks okay to me, on reflection.  Am I missing some issue here?

**Reply by harshalbhatt23** [reply]
We can split verses when chapter look more complicate and many changes in verses while transcribing. But to do this we have to glance the whole chapter in MS to see if it is needed or not. 
Btw you explained the issued very well.....thank you

**Comment by chchch** [comment]
Instead of `<l xml:id="SS.5.3.35.2" sameAs="#SS.5.3.39.2">`, why not just write `<l xml:id=SS.5.3.39.2">` ?

**Reply by chchch** [reply]
Then it will get collated against all other versions of that verse with the same `@xml:id`.

**Comment by wujastyk** [comment]
Because you can't use the same xml:id twice.  The XML won't validate.

**Comment by chchch** [comment]
Ah I see, so that witness actually has the same verse twice?

**Comment by wujastyk** [comment]
In some cases, yes
